src/components/Board.js

```diff
@@ -5,16 +5,30 @@ import PropTypes from 'prop-types';
 
 
 const generateSquareComponents = (squares, onClickCallback) => {
+  console.log(onClickCallback)
   // Complete this for Wave 1
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
 
+  let newArr = [];
+  for (let i = 0; i < squares.length; i++) {
+    newArr = newArr.concat(squares[i]);
+  }
+
+  let squaresArr = [];
+  for (let i = 0; i < newArr.length; i++) {
+    const currentSquare = newArr[i];
+    squaresArr.push(<Square value={currentSquare.value} id={currentSquare} onClickCallback={onClickCallback} />)
```

Comment:
You are failing 2 tests because `id` is an object rather than a number. 

Maybe you meant to do the following?: 

```suggestion
    squaresArr.push(<Square value={currentSquare.value} id={currentSquare} onClickCallback={onClickCallback.id} />)
```